terraform/applications/careers/static.tf

```diff
@@ -0,0 +1,189 @@
+# all the relevant infra for the static site
+resource "aws_s3_bucket" "logs" {
+  bucket = "mozilla-careers-logs-783633885093"
+  acl    = "log-delivery-write"
+}
+
+resource "aws_s3_bucket" "mozilla-careers" {
+  bucket = local.bucket_name
+  acl    = "log-delivery-write"
+
+  force_destroy = true
+
+  hosted_zone_id = local.s3_zone
+  logging {
+    target_bucket = aws_s3_bucket.logs.bucket
+    target_prefix = "stage-logs/"
+  }
+
+  website {
+    index_document = "index.html"
+  }
+
+  website_domain   = local.s3_website_domain
+  website_endpoint = local.s3_website_endpoint
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Id": "careers policy",
+  "Statement": [
+    {
+      "Sid": "careersAllowListBucket",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:ListBucket",
+      "Resource": "arn:aws:s3:::${local.bucket_name}"
+    },
+    {
+      "Sid": "careersAllowIndexDotHTML",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:GetObject",
+      "Resource": "arn:aws:s3:::${local.bucket_name}/*"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_acm_certificate" "careers_cert" {
+  provider                  = aws.aws-east
+  domain_name               = local.domain_name
+  subject_alternative_names = local.subject_alternative_names
+  validation_method         = "DNS"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_cloudfront_distribution" "s3_distribution" {
+  origin {
+    domain_name = local.s3_website_endpoint
+    origin_id   = "mozilla-careers"
+    custom_origin_config {
```

Comment:
Is this a place we can use s3_origin_config {} instead of handcrafting it? Brett and I looked into this on another PR recently and he seemed positive about it.

Reply:
@bkochendorfer feel free to suggest here, this was just me porting the old pattern

Reply:
You get slightly different behavior when using custom_origin cs s3_origin_config. For my use case we looked at the s3_origin_config version but preferred the behavior of the custom_origin config. custom_origin config acts more like a website, a request to the root will default to index.html for example. For s3_config the request to root will list all the objects in the bucket.

Reply:
Aha, interesting. Thanks for checking into that.

Reply:
Sounds like the right choice in this case is custom_origin, since this is a website.  Thanks for clarifying, I'll leave it as-is.